packaging/os/pkgutil.py

```diff
@@ -63,10 +63,10 @@
 import pipes
 
 def package_installed(module, name):
-    cmd = [module.get_bin_path('pkginfo', True)]
+    cmd = ['pkginfo']
```

Comment:
ah, i stopped at this change, i see what you are doing in later in the local run_command, the original run_command does not make much sense to me either, but this seems like a smaller change over adding the extra search path to each one.